lib/Sources_process.py

```diff
@@ -221,6 +221,30 @@ def process_cve_item(self, item=None):
         if "impact" in item:
             cve["access"] = {}
             cve["impact"] = {}
+            cve['impact3'] = {}
+            cve['exploitability3'] = {}
```

Comment:
These 2 fields should only be set if baseMetricV3 is present; so I would move them to the if statement checking for that specific field in item['impact']